### ISSUE TITLE
Typo in your Dockerfile and in your Dockerhub Image

### ISSUE THREAD

**Issue by MephistoJB** [Sabuto/hassio-nextcloud-dev#1]
It seems you have a typo in your Dockerfile. You wrote "ehco" instead of "echo". Same in your Image. There in your php config folder in the acmu file. Great work though ;)

**Comment by sabuto** [comment]
Thanks for this :), i stopped working on this for a while as it was annoying me that it wasn't working. Thanks for this actions building now so fingers crossed :)

**Comment by MephistoJB** [comment]
You are welcome.

I am wondering. Did you take into consideration to take a nextcloud docker image and just wrap it into a container for hassio? Or is there an advantage in your approach. You have to apologize, but I am very new in hassio.

**Comment by sabuto** [comment]
So I didnt really get a chance to look at the next cloud one properly but I think they use Apache and not nginx. This is basically a wrapper for linuxserver docker image for next cloud. But when I get time I will re look at the official next cloud oneOn 25 Apr 2020 12:40 pm, MephistoJB <notifications@github.com> wrote:
You are welcome.
I am wondering. Did you take into consideration to take a nextcloud docker image and just wrap it into a container for hassio? Or is there an advantage in your approach. You have to apologize, but I am very new in hassio.

**Comment by MephistoJB** [comment]
Hi @Sabuto I found another repo, where someone did this. Already forked it, to upgrade and I am in Contact with the owner. Maybe you want to join us? Could need some help to integrate it into ingress. Look [here](https://github.com/mtthp/hassio-addons)

**Comment by sabuto** [comment]
> Hi @Sabuto I found another repo, where someone did this. Already forked it, to upgrade and I am in Contact with the owner. Maybe you want to join us? Could need some help to integrate it into ingress. Look [here](https://github.com/mtthp/hassio-addons)

I will have a look at this indie not know this existed. I will look into forking and getting ingress sorted :) saves me a lot of work :)

**Comment by MephistoJB** [comment]
Yes, good idea. I am trying to config it to use the MariaDB add-on at the moment. Did you do something like this in the past? Though I could just copy and past from the book stack add-on, but this doesn’t seem to that easy :-D
